src/daemons/server-stats.ts

```diff
@@ -1,5 +1,5 @@
 import * as os from 'os';
-const osUtils = require('os-utils');
+const sysUtils = require('systeminformation');
```

Comment:
I'm feeling the below code is better than that.
```js
import * as sysUtils from 'systeminformation';
```

Reply:
Done